testsuite/mailhog.py

```diff
@@ -80,3 +118,19 @@ def delete(self, mail_id=None):
         else:
             for mail in mail_id:
                 self.request(method="DELETE", endpoint=f"api/v1/messages/{mail}")
+
+    @backoff.on_exception(backoff.fibo, AssertionError, max_tries=8, jitter=None)
+    def assert_message_received(self, expected_count=1, subject=None, content=None):
```

Comment:
Specify if `subject` and `content` are mutually exclusive or not, as now result of find_by_subject would be overwritten and not checked.